manager/src/main/scala/org/labrad/manager/auth/AuthStore.scala

```diff
@@ -135,10 +138,14 @@ class AuthStoreImpl(
   }
 
   def checkUserPassword(username: String, password: String): Boolean = synchronized {
-    val hashed = SQL"""
-      SELECT password_hash FROM users WHERE name = $username AND password_hash IS NOT NULL
-    """.as(get[String]("password_hash").singleOpt)
-       .getOrElse { sys.error(s"username $username does not exist or has no password") }
+    val hashed = if (username.isEmpty) {
+      globalPasswordHash
```

Comment:
Do you think it'd be useful to have an error message that reflects the assumption that a blank username will fall back to the global password? e.g. in case the user relies on env vars and is in the wrong env, or forgets the right combination of arguments.